fix technologies download

afreudenreich opened the pull request:
Hi Ross,

If you like you can have a look at my take at this - I tested it briefly and it seems to work again. Might need some more QA.

Thanks,
Andreas

afreudenreich committed:
fix technologies download

RossGeerlings commented:
Thanks, I will probably take at least parts of this with some changes.  I'll put a credit to you in the readme and source if you like.  Do you have a preference on how I list your name?

afreudenreich commented:
Thanks Ross - no need for the credit, just a few lines of code :)
You did the hard work before...

> On Sep 10, 2021, at 1:24 PM, Ross Geerlings ***@***.***> wrote:
> 
> 
> Thanks, I will probably take at least parts of this with some changes. I'll put a credit to you in the readme and source if you like. Do you have a preference on how I list your name?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/RossGeerlings/webstor/pull/3#issuecomment-917189313>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABHFZK5OV7KCYF57OY4W5YTUBJSO7ANCNFSM5DZ6IGCQ>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>

afreudenreich committed:
add technologies to DB in json format
remove formatting from search_wappalyzer
fetch only one technology
allow lists in meta object

afreudenreich commented:
Hi Ross,
I added another commit, with several smaller changes. One of them saves the technologies directly as json into the MySQL DB, so you can drop all the complex single quote replacements. With this change I can run a search against all technologies without getting an error. Feel free to use.
Thanks - Andreas
